examples/docs/en-US/date-picker.md

```diff
@@ -64,7 +64,11 @@
         value8: '',
         value9: '',
         value10: '',
-        value11: ''
+        value11: '',
+        value12: '',
+        value13: '',
+        defaultTimeWithMin: [new Date(2018, 1, 1, 12)],
```

Comment:
It will be easier for users to provide time string instead of `Date` parsable object. But it will introduce a new format of time data and parse behavior, which means extra API & realization complexity. I'm not sure if it deserves that.

I found another problem of this PR: it is impossible to configure millisecond value of `defaultTime` now (because I'm using `modifyTime`). I will fix that.

Reply:
I think fecha's parse method can handle time alone. (dateUtil in 'date-picker/util/index.js')
You can also refer to time-select panel's parseTime.

Reply:
I don't think millisecond value is an issue since it is not respected in any picker panels (They may reset millisecond to 0).

Also I don't think there are compelling use cases for it. For example, using half-open intervals link `[00:00:00.000, +1 days 00:00:00.000)` instead of closed interval `[00:00:00.000, 23:59:59.999]` eliminates the need of millisecond values.

Reply:
Thanks, I'll check it.